lessons/08-html-site/index.html

```diff
@@ -0,0 +1,28 @@
+<html>
+    <head>
+        <title> Home | Rachna Jain</title>
+    </head>
+    <body>
+        <header>
+            <img class ="fit picture" src="https://images.unsplash.com/photo-1494790108377-be9c29b29330?ixlib=rb-0.3.5&q=80&fm=jpg&crop=entropy&cs=tinysrgb&w=200&fit=max&s=707b9c33066bf8808c934c8ab394dff6" alt="face image" width="200" height="200">
+            <h2>Rachna's Portfolio</h2>
+            <nav>
+                <a href="index.html">Home</a>
+                <a href="about.html">About</a>
+                <a href="contact.html">Contact</a>
+            </nav>
+            <hr>
+         </header>
+         <p>
+                <h1><strong>Home</strong></h1>
+            </p>
+            <p>
+                Welcome to my portfolio.click "About" to learn more about me.
+           </p>
+           <form action="https://google.ca/search" method="GET">
+            <label>Search</label>   
+            <input type="text" name="search" pattern="q*" placeholder="name must start with q">
```

Comment:
Close! What i meant by "name must be q" is that the name **attribute** here has to be set to "q" otherwise google wont know what you sent it

So here it'd be

```html
<input type="text" name="q" />
```

But I'm impressed you found this pattern attribute!